qf_lib/backtesting/portfolio/portfolio.py

```diff
@@ -64,6 +66,21 @@ def __init__(self, data_handler: DataHandler, initial_cash: float, timer: Timer)
 
         self.logger = qf_logger.getChild(self.__class__.__name__)
 
+    def _current_exchange_rate(self, currency: str) -> float:
+        """Last available exchange rate from the specified currency to the portfolio currency."""
+        if currency == self.currency:
+            return 1.
+        return self.data_handler.get_last_available_exchange_rate(currency, self.currency, frequency=Frequency.DAILY)
```

Comment:
DataHandlers have builtin frequencies (DailyDataHandles has DAILY etc), so I believe it would be possible to omit the frequency here in the function call and take it from the data handler